app/controllers/instrument_request_approval_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InstrumentRequestApprovalController < ApplicationController
```

Comment:
Now that I see what this controller is doing, the name seems odd. To me, this  seems like it should be doing the approval, not the request.